source/Camera/InfoUtilOther/ZividBenchmark/ZividBenchmark.cpp

```diff
@@ -459,6 +472,82 @@ namespace
         printCapture2DResults(allDurations);
     }
 
+    Zivid::Frame assistedCapture(Zivid::Camera &camera)
+    {
+        const auto parameters = Zivid::CaptureAssistant::SuggestSettingsParameters{
+            Zivid::CaptureAssistant::SuggestSettingsParameters::AmbientLightFrequency::none,
+            Zivid::CaptureAssistant::SuggestSettingsParameters::MaxCaptureTime{ std::chrono::milliseconds{ 800 } }
+        };
+        const auto settings = Zivid::CaptureAssistant::suggestSettings(camera, parameters);
+        return camera.capture(settings);
+    }
+
+    template<typename DataType>
+    Duration copyDataTime(Zivid::Frame frame, DataType copyName)
+    {
+        std::vector<Duration> copyDataDurations;
+
+        auto pointCloud = frame.pointCloud();
+        const auto beforeCopyData = HighResClock::now();        
+        const auto tempCopy = pointCloud.copyData<decltype(copyName)>();
```

Comment:
You don't need `tempCopy` I think. Maybe just leave `pointCloud.copyData<decltype(copyName)>();` instead of `const auto tempCopy = pointCloud.copyData<decltype(copyName)>();`